ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -972,59 +968,6 @@ def verify_ocs_csv(ocs_registry_image=None):
             )
 
 
-@retry(AssertionError, 60, 10, 1)
```

Comment:
We will probably need to keep it for deployment of 4.18 for upgrade. Or as you mentioned on our 1:1 after build -59 we will not check it.

Reply:
done